### ISSUE TITLE
A single broken Stack or StackSet shouldn't affect processing of other Stacks

### ISSUE THREAD

**Issue by linki** [zalando-incubator/stackset-controller#48]
A faulty `stackset.yaml` (we don't validate the content yet) led to this in the logs:

```console
ERROR: logging before flag.Parse: E0904 16:04:08.607962       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:09.635793       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:10.656678       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:11.686808       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:12.693538       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:13.702829       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:14.714000       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:15.767220       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:16.771295       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:17.774734       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:18.779158       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:19.787391       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
ERROR: logging before flag.Parse: E0904 16:04:20.803426       1 reflector.go:205] github.com/zalando-incubator/stackset-controller/controller/stackset.go:482: Failed to list *v1.StackSet: v1.StackSetList.Items: []v1.StackSet: v1.StackSet.Spec: v1.StackSetSpec.StackTemplate: v1.StackTemplate.Spec: v1.StackSpecTemplate.StackSpec: Service: v1.StackServiceSpec.Ports: []v1.ServicePort: v1.ServicePort.Port: readUint32: unexpected character: �, error found in #10 byte of ...|","port":"80","proto|..., bigger context ...|cas":2,"service":{"ports":[{"name":"http","port":"80","protocol":"TCP","targetPort":"8080"}]},"versi|...
```

**Comment by mikkeloscar** [comment]
Hmm, great, breaks in the watch.... Depends on #10

**Comment by mikkeloscar** [comment]
Fixed in #52
